From: Sebastian Jennen <sebastian.jennen@gmx.de>
Date: Thu, 13 Feb 2020 20:15:13 +0100
Subject: sockopt autocreate cleanup

- add info to README about generate.sh script
- changed sockopts_set.gsl to honor zend_long over long
- changed year to 2020
- (also ran gsl -script:sockopts.gsl sockopts.xml which has no impact on zmq_sockopt.c now)
---
 zmq-1.1.3/zmq_sockopt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zmq-1.1.3/zmq_sockopt.c b/zmq-1.1.3/zmq_sockopt.c
index 3a00421..fc9949b 100644
--- a/zmq-1.1.3/zmq_sockopt.c
+++ b/zmq-1.1.3/zmq_sockopt.c
@@ -1,7 +1,7 @@
 /*
 +-----------------------------------------------------------------------------------+
 |  ZMQ extension for PHP                                                            |
-|  Copyright (c) 2010-2016, Mikko Koppanen <mkoppanen@php.net>                      |
+|  Copyright (c) 2010-2020, Mikko Koppanen <mkoppanen@php.net>                      |
 |  All rights reserved.                                                             |
 +-----------------------------------------------------------------------------------+
 |  Redistribution and use in source and binary forms, with or without               |
